assets/js/main.js

```diff
@@ -8553,33 +8554,36 @@ $(function() {
     $("#error-message").html("");
     form.find("*").removeClass("has-error");
   };
-  
+
   var submitForm = function(e) {
     e.preventDefault();
     if (inCall) {
       return;
     }
     inCall = true;
 
+    var url = form.prop('action');
     var data = serialize(form.get(0), { hash: true });
+    var redirect = next.val();
+
     if (validateForm()) {
       grecaptcha.ready(function() {
         grecaptcha.execute('6LcXFLoZAAAAAHVaImPgU3xGnBmyY-lwQ6sHllGN', {action: 'submit'}).then(function(token) {
           data['g-recaptcha-response']=token;
-          submitToFormSpree(data);
+          submitToFormSpree(url, data, redirect);
         });
       });
     } else {
       inCall = false;
     }
   };
 
-  var submitToFormSpree = function(data) {
+  var submitToFormSpree = function(url, data, redirect) {
     submitButton.html("Sending...");
     submitButton.prop("disabled", true);
 
     var postParams = {
-      url: "https://api.formbucket.com/f/buk_7iB8j7vEJPW9ad2ClJwFfm5M",
+      url: url,
```

Comment:
I've updated our form submission code to read the URL from the `action` of the form, so we can use the same code with multiple forms. The redirect URL is configurable too, via the `_next` param.

Reply:
Cool!